tslint.json

```diff
@@ -44,7 +44,7 @@
       }
     ],
     "no-console": [
-      true,
+      false,
```

Comment:
Pq essa alteração? Se não for necessária, pode remover?

Reply:
Aqui foi para permitir a exibição de console.log para debugar o código mais facilmente nas previews do netlify, é para reverter mesmo antes de subir o PR.